### ISSUE TITLE
Settings are not saved and are cleared when phone turns off or reset ram

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ionux/genplus-droid#30]
```
What steps will reproduce the problem?
1. Set your settings for Genplusdroid when you exit your settings are saved
2. When you turn off the phone and turn back on the custom settings are reset 
back to defaults
3. When you clean the ram it too will clear and reset settings back to default

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
1.9.2

Please provide any additional information below.
I sent an email that details this better

```

Original issue reported on code.google.com by `segafa...@gmail.com` on 30 Mar 2012 at 11:00

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.9.3.  Just released.
```

Original comment by `shinhals...@gmail.com` on 2 Apr 2012 at 11:02

**Comment by GoogleCodeExporter** [comment]
Original comment by `shinhals...@gmail.com` on 2 Apr 2012 at 11:02
- Changed state: **Fixed**
